pypodcastparser/Item.py

```diff
@@ -261,23 +297,69 @@ def set_published_date(self, tag):
             if len(deconstructed_date) < 4:
                 raise AttributeError
 
+            # Map of timezone offsets to timezone abbreviations
```

Comment:
can you add some unit tests for the new logic

Reply:
Done. There was no new logic added, just mainly some additional data, so i added a test for an offset that started with 0, just in case.